src/Og.php

```diff
@@ -413,12 +413,10 @@ public static function getGroupCount(EntityInterface $entity, $group_type_id = N
   public static function getGroupContentIds(EntityInterface $entity, array $entity_types = []) {
     $group_content = [];
 
+
     // Retrieve the fields which reference our entity type and bundle.
     $query = \Drupal::entityQuery('field_storage_config')
-      // @todo For the moment retrieving both group types, since there seems to
-      //   be some confusion about which field type is used for users.
-      // @see https://github.com/amitaibu/og/issues/177
-      ->condition('type', ['og_standard_reference', 'og_membership_reference'], 'IN');
+      ->condition('type', OgGroupAudienceHelper::NON_USER_REFERENCE_FIELD);
```

Comment:
as we now separate user and non-user logic, the query here should only bring non-user entities.